onnxruntime/core/optimizer/skip_layer_norm_fusion.cc

```diff
@@ -12,7 +12,7 @@ using namespace onnxruntime::common;
 namespace onnxruntime {
 
 // LayerNorm supports limited data types.
-static std::vector<std::string> supported_data_types{"tensor(float16)", "tensor(float)", "tensor(bfloat16)"};
+static constexpr std::array<const char*, 3> supported_data_types{"tensor(float16)", "tensor(float)", "tensor(bfloat16)"};
```

Comment:
This change is to change a runtime initialization to compile time, to avoid initialization order problems.